features/resize.yml

```
@@ -0,0 +1,11 @@
+name: Resize
+description: The `resize` CSS property sets whether an element can be resized by the user, and on which axes.
+spec: https://drafts.csswg.org/css-ui-4/#resize
+caniuse: css-resize
+group: css
+# Baseline is false due to no Safari on iOS support, but it is available on all non-mobile browsers.
+compat_features:
```

Since we're using `caniuse: css-resize`, we have an obligation to set the headline status so we line up (at least roughly) with it. It doesn't make a huge difference in a non-Baseline case like this, but this would line us up with caniuse's Firefox 5:

```suggestion
status:
  compute_from: css.properties.resize.block_level_support
compat_features:
```